composer.json

```diff
@@ -22,8 +22,8 @@
     "homepage": "https://github.com/ipinfo/ipinfolaravel",
     "keywords": ["Laravel", "ipinfolaravel"],
     "require": {
-        "illuminate/support": ">=5, <10",
-        "ipinfo/ipinfo": "^2.3.1"
+        "illuminate/support": ">=5",
+        "ipinfo/ipinfo": ">=2.3.1"
```

Comment:
This is not safe to say `>=` and nor is it safe to say `>=5` on `illuminate/support`. These things must be bounded